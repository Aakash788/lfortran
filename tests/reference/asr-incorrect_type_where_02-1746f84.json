{
    "basename": "asr-incorrect_type_where_02-1746f84",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/incorrect_type_where_02.f90",
    "infile_hash": "4fbb3a440309d31eb008cd3fffcf541729477e6ac4fc23854bfa1f00",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-incorrect_type_where_02-1746f84.stderr",
    "stderr_hash": "9cc684763c7a835daede875fc12bbdb892b9a512320eb7739745bfc2",
    "returncode": 2
}