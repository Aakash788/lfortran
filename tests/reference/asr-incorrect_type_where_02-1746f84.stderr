semantic error: 'where' clause requires a logical array
 --> tests/errors/incorrect_type_where_02.f90:5:10
  |
5 |    where(1) b = 12121
  |          ^ 
